### PR TITLE
Add Namibia field boundaries_2022

### PR DESCRIPTION
**Pull request by MsPixels**
These labels were sampled from the field boundaries and will be used to generate an in-season  (Sept - Jan 2023) map for Namibia

### CODE REVIEW

File: data/raw.dvc [hunk 1]
```diff
@@ -1,4 +1,6 @@
 outs:
+- md5: a532c2e43fdb5903eecfa5d4f1601582.dir
+  size: 438914978
```
Comment (ivanzvonkov) [inline data/raw.dvc:3]: It looks like this was merged with "accept both" which is not what we want here. Merging dvc files is difficult and I haven't written instructions for that yet. So in the meantime I recommend going through the dvc resources and Github merging in https://github.com/nasaharvest/crop-mask/wiki/2.-Getting-Started

Comment (MsPixels) [reply data/raw.dvc]: Hmmm, alrightyy, I'll take a look. Thanks

**Comment by MsPixels** [comment]
Gcloud permission denied
image

**Comment by ivanzvonkov** [comment]
Updated start date to 2021 and permissions on Google Cloud and looks like it's working

**Review by ivanzvonkov — APPROVED** [review]

**Pull request merged**